test-data/src/main/resources/org/openmrs/module/fhir2/providers/ObservationWebTest_create.json

```diff
@@ -1,22 +1,22 @@
-{
-  "resourceType": "Observation",
-  "status": "preliminary",
-  "code": {
-    "coding": [
-      {
-        "code": "5090AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
-      }
-    ]
-  },
-  "subject": {
-    "reference": "Patient/5946f880-b197-400b-9caa-a3c661d23041"
-  },
-  "encounter": {
-    "reference": "Encounter/6519d653-393b-4118-9c83-a3715b82d4ac"
-  },
-  "effectiveDateTime": "2014-11-25T22:17:00+11:00",
-  "valueQuantity": {
-    "value": 156,
-    "unit": "cm"
-  }
-}
+{
+  "resourceType": "Observation",
+  "status": "preliminary",
+  "code": {
+    "coding": [
+      {
+        "code": "5090AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+      }
+    ]
+  },
+  "subject": {
+    "reference": "Patient/5946f880-b197-400b-9caa-a3c661d23041"
+  },
+  "encounter": {
+    "reference": "Encounter/6519d653-393b-4118-9c83-a3715b82d4ac"
+  },
+  "effectiveDateTime": "2014-11-25T22:17:00+11:00",
+  "valueQuantity": {
+    "value": 156,
+    "unit": "cm"
+  }
+}

```

Comment:
I'm not sure why this file was changed. Can you check if you have Git's autocrlf feature enabled or not?

Reply:
i have enabled that using `git config --global core.autocrlf true` (since I am working on windows 11). Then gone ahead to update all files on this branch to reflect the new configuration using the commands below: 
```
git rm -rf --cached .
git reset --hard HEAD
```